.pre-commit-config.yaml

```diff
@@ -8,15 +8,6 @@ repos:
         -   id: check-yaml
         -   id: end-of-file-fixer
         -   id: trailing-whitespace
-    -   repo: https://github.com/psf/black-pre-commit-mirror
```

Comment:
I am okay removing isort. However, I am not sure about black. Do the formatters conflict?

Reply:
The Ruff formatter is a drop-in replacement for Black, but much faster (100x). According to [these docs](https://docs.astral.sh/ruff/formatter/#ruff-format) "> 99.9% of lines are formatted identically", so this may explain some minor differences of formatting found in this PR.

Reply:
Sounds worth a try. Cand always swap it back later. Probably should remove the black badge in the readme.